README.md

```diff
@@ -122,7 +122,9 @@ These largely correspond to the underscored option keys for both library interfa
     "break_chained_methods": false,
     "eval_code": false,
     "unescape_strings": false,
-    "wrap_line_length": 0
+    "wrap_line_length": 0,
+    "wrap_attributes": "auto",
+    "wrap_attributes_indent_size": 4
```

Comment:
This should default to `indent_size`.  For example, if `indent_size`is set to 2, and this is not set, it should default to 2.